samcli/local/lambda_service/lambda_error_responses.py

```diff
@@ -0,0 +1,89 @@
+""" Common Lambda Error Response"""
+
+import json
+
+from flask import Response
+
+
+class LambdaErrorResponses(object):
+
+    # The content type of the Invoke request body is not JSON.
+    UnsupportedMediaTypeException = ('UnsupportedMediaType', 415)
+
+    # The AWS Lambda service encountered an internal error.
+    ServiceException = ('Service', 500)
+
+    # The resource (for example, a Lambda function or access policy statement) specified in the request does not exist.
+    ResourceNotFoundException = ('ResourceNotFound', 404)
+
+    # The request body could not be parsed as JSON.
+    InvalidRequestContentException = ('InvalidRequestContent', 400)
+
+    @staticmethod
+    def resource_not_found(function_name):
+        """
+
+        Parameters
+        ----------
+        function_name
+
+        Returns
+        -------
+
+        """
+        return LambdaErrorResponses._service_response(json.dumps(
```

Comment:
All the responses below share the same structure, but I already see some typos - `Message` property is capitalized here but not in the `invalid_request_context` method. 

Can you instead create one method that takes in the msg and error enum (ex: UnsupportedMediaTypeException) and returns you a service response? That way you have one place where the key names are written down.